contracts/FractalModule.sol

```diff
@@ -45,10 +49,7 @@ contract FractalModule is IFractalModule, Module {
             bytes memory _data,
             Enum.Operation _operation
         ) = abi.decode(execTxData, (address, uint256, bytes, Enum.Operation));
-        require(
-            exec(_target, _value, _data, _operation),
-            "Module transaction failed"
-        );
+        if(!exec(_target, _value, _data, _operation)) revert TxFailed();
```

Comment:
if the `exec` function fails here can it even continue anyway? wouldn't `exec` revert?

Reply:
Nope, `exec` will return `true` or `false` depending on whether the transaction fails or succeeds, but shouldn't revert. 

https://github.com/gnosis/zodiac/blob/master/contracts/core/Module.sol#L37